### PR TITLE
fix: plan usage with overages

### PR DESCRIPTION
**Pull request by gris**
Screen Shot 2023-09-22 at 11 21 15 AM
Screen Shot 2023-09-22 at 11 21 32 AM

https://github.com/tursodatabase/turso-cli/issues/646

### CODE REVIEW

File: internal/cmd/plan.go [hunk 1]
```diff
@@ -55,36 +55,63 @@ var planShowCmd = &cobra.Command{
...
+		if err != nil {
+			return err
+		}
+		var currentOrg turso.Organization
+		for _, org := range orgs {
+			if org.Slug == organizationName {
+				currentOrg = org
+				break
+			}
+		}
```
Comment (athoscouto) [inline internal/cmd/plan.go:68]: ⛏️: move this into `func getCurrentOrg`

File: internal/cmd/plan.go [hunk 2]
```diff
@@ -55,36 +55,63 @@ var planShowCmd = &cobra.Command{
...
 		fmt.Printf("\nQuota will reset on %s\n", getFirstDayOfNextMonth().Local().Format(time.RFC1123))
 		return nil
 	},
 }
 
+func overagesMessage(overages bool) string {
+	if overages {
+		return fmt.Sprintf("Overages %s\n", internal.Emph("enabled"))
+	}
+	return fmt.Sprintf("Overages %s\n", internal.Emph("disabled"))
```
Comment (athoscouto) [inline internal/cmd/plan.go:88]: ⛏️⛏️⛏️ 
```suggestion
	status := "disabled"
	if overages {
		status = "enabled"
	}
	return fmt.Sprintf("Overages %s\n", internal.Emph(status))
```

File: internal/cmd/plan.go [hunk 3]
```diff
@@ -55,36 +55,63 @@ var planShowCmd = &cobra.Command{
...
+		for _, org := range orgs {
+			if org.Slug == organizationName {
+				currentOrg = org
+				break
+			}
+		}
 
 		fmt.Printf("Organization: %s\n", internal.Emph(organizationName))
-
+		currentOrg.Overages = true
```
Comment (athoscouto) [inline internal/cmd/plan.go:71]: Is this for debugging?

**Review by athoscouto — APPROVED** [review]

**Review by avinassh — APPROVED** [review]

File: internal/cmd/plan.go (hunk 3 above)
Comment (gris) [reply internal/cmd/plan.go:71]: Yep! removing it

**Pull request merged**